### PR TITLE
JAVA-1908: TableMetadata.asCQLQuery does not add table option 'memtable_flush_period_in_ms' in the generated query

### PR DESCRIPTION
**Pull request by hkbharath**
Adding memtable_flush_period_in_ms option in table metadata returned from asCQLQuery from tableMetaData.

### CODE REVIEW

**Comment by hkbharath** [comment]
How can i get this change reviewed? Please help

File: driver-core/src/main/java/com/datastax/driver/core/AbstractTableMetadata.java [hunk 1]
```diff
@@ -280,6 +280,7 @@ protected StringBuilder appendOptions(StringBuilder sb, boolean formatted) {
         if (cassandraVersion.getMajor() > 3 || (cassandraVersion.getMajor() == 3 && cassandraVersion.getMinor() >= 8)) {
             and(sb, formatted).append("cdc = ").append(options.isCDC());
         }
+		and(sb, formatted).append("memtable_flush_period_in_ms = ").append(options.getMemtableFlushPeriodInMs());
```
Comment (adutra) [inline driver-core/src/main/java/com/datastax/driver/core/AbstractTableMetadata.java]: This option has been introduced in Cassandra 2.0 by CASSANDRA-4237.

So i would suggest that you only include it when the version is 2.0 or higher:

```
        if (cassandraVersion.getMajor() > 1) {
            and(sb, formatted).append("memtable_flush_period_in_ms = ").append(options.getMemtableFlushPeriodInMs());
        }
```

**Review by adutra — CHANGES_REQUESTED** [review]
Please address my comment and also add a test for your changes in `com.datastax.driver.core.TableMetadataTest#should_parse_table_options`. Thanks!

**Comment by adutra** [comment]
Also, as per our [contribution guidelines](https://github.com/datastax/java-driver/blob/3.x/CONTRIBUTING.md) could you please add an entry to the changelog for this? Thanks!

File: driver-core/src/main/java/com/datastax/driver/core/AbstractTableMetadata.java (hunk 1 above)
Comment (hkbharath) [reply driver-core/src/main/java/com/datastax/driver/core/AbstractTableMetadata.java]: Thanks for the suggestion. Added suggested changes.

**Comment by tolbertam** [comment]
Changed this to 3.6.0 as I think it's reasonable to include in 3.6.0 as this is a bugfix and we still have time before releasing 3.6.0.

**Review by tolbertam — APPROVED** [review]
Looks great to me, thanks! :+1:

**Review by adutra — APPROVED** [review]

**Comment by adutra** [comment]
@hkbharath I fixed a few small nits manually, squashed and rebased your changes on top of 3.x as 5fa08fba41ec031713b2a7a4841245b3d689cbde. Thank you for your contribution, highly appreciated! 👍

**Comment by hkbharath** [comment]
Thanks @adutra .

**Pull request closed without merging**